cmd/root.go

```diff
@@ -144,15 +142,21 @@ func runE(v *viper.Viper, statz *stats.Stats, cmd *cobra.Command, args []string)
 	log.SetOutput(os.Stderr)
 	log.SetReportTimestamp(false)
 
+	// set log level based on verbosity
+	// by default we only emit errors
 	switch v.GetInt("verbose") {
 	case 0:
-		log.SetLevel(log.WarnLevel)
+		log.SetLevel(log.ErrorLevel)
 	case 1:
+		log.SetLevel(log.WarnLevel)
+	case 2:
 		log.SetLevel(log.InfoLevel)
 	default:
 		log.SetLevel(log.DebugLevel)
 	}
 
+	log.Infof("using config file: %s", configFile)
```

Comment:
This could be a debug statement since it's only useful to debug things out.

Reply:
Made this change